riff-raff.yaml

```diff
@@ -19,10 +19,10 @@ deployments:
       prependStackToCloudFormationStackName: false
       amiParametersToTags:
         AMITyperighterchecker:
-          Recipe: editorial-tools-focal-java11-ngrams-ARM-cdk-base
+          Recipe: editorial-tools-jammy-java11-ngrams-ARM-cdk-base
           BuiltBy: amigo
         AMITyperighterrulemanager:
-          Recipe: editorial-tools-focal-java11-ARM-WITH-cdk-base
+          Recipe: editorial-tools-jammy-java11
```

Comment:
I have used `editorial-tools-jammy-java11` instead of `editorial-tools-jammy-java11-ARM-WITH-cdk-base`, which is identical except it doesn't included the unrecommended[ await tags role](https://amigo.gutools.co.uk/roles#aws-await-tags).

Reply:
I checked the code and it appears not to be using tags to determine its stage.